src/resources/addys/claimables/types.ts

```diff
@@ -46,22 +46,45 @@ export enum ClaimableType {
   HypersubV2 = 'hypersub-v2',
   Clanker = 'clanker',
   Rainbow = 'rainbow',
+  RainbowEthRewards = 'rainbow-eth-rewards',
   RainbowSuperTokenCreatorFees = 'rainbow-super-token-creator-fees',
 
   // test claims
   Testing = 'frontend-testing',
   TestingSponsored = 'frontend-testing-sponsored',
 }
 
+export interface ClaimableAsset<T, A> {
+  asset: T;
+  amount: A;
+  usd_value: number;
+  value: string;
+}
```

Comment:
different asset types and amount can be formatted or a raw string depending on where we are in the flow.